### PR TITLE
Pass `Request` to EntityValueResolver

### LINKED ISSUE #18987: [DoctrineBridge] Pass `Request` to `EntityValueResolver`'s expression
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#51800
| PR author(s) | @HypeMC
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/18987

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -746,6 +746,23 @@ In the example above, the ``$product`` argument is handled automatically,
...
 
+If you need to get other information from the request to fetch your entity, you
+can also access to the request in your expression thanks to the ``request``
+variable. Let's say you pass the page limit of a list in a query parameter::
+
+    #[Route('/product/{id}/comments')]
+    public function show(
+        Product $product,
+        #[MapEntity(expr: 'repository.findBy(["product_id" => id], null, request.query.get("limit", 10)')]
+        Comment $comment
```
Comment (94noni) [inline doctrine.rst]: shouldn'it be an array/collection ? due to the `findBy`

Comment (alexandre-daubois) [reply doctrine.rst]: That's right, thanks!

File: doctrine.rst [hunk 2]
```diff
@@ -746,6 +746,23 @@ In the example above, the ``$product`` argument is handled automatically,
 but ``$comment`` is configured with the attribute since they cannot both follow
 the default convention.
 
+If you need to get other information from the request to fetch your entity, you
```
Comment (94noni) [inline doctrine.rst]: nitpick, but then this should be your entities, or at least your data ^^

**Review by 94noni — APPROVED** [review]

File: doctrine.rst (hunk 2 above)
Comment (alexandre-daubois) [reply doctrine.rst]: Reworded to match any case :ok_hand:

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Comment by seb-jean** [comment]
@alexandre-daubois I can't get your example to work. I encounter an error:
`Controller "App\Controller\Admin\BlogController::index" requires that you provide a value for the "$authorPosts" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or there is a non-optional argument after this one.`

**Pull request merged**